agate/agate/views.py

```diff
@@ -19,6 +19,33 @@ def ingestion_attempt_response(request, project=""):
     return JsonResponse(data, safe=False)
 
 
+def single_ingestion_attempt_response(request, uuid=""):
+    try:
+        obj = IngestionAttempt.objects.get(uuid=uuid)
+    except IngestionAttempt.DoesNotExist:
+        return HttpResponse('Unauthorized', status=status.HTTP_404_NOT_FOUND)
+
+    auth = request.headers.get("Authorization")
+    if (not check_authorized(auth, obj.site, obj.project)):
```

Comment:
This isn't wrong but Python doesn't need one set of brackets:
```suggestion
    if not check_authorized(auth, obj.site, obj.project):
```